karma.conf.ts

```diff
@@ -0,0 +1,14 @@
+module.exports = function (config: any) {
```

Comment:
you barely use any ts features here, does it need to be ts and not js?

Reply:
ehhh probably not. I guess I just wanted to be uniform with the project lol

Reply:
Hi sir, I tried to update to JS and basically had to go back to commonJS "export default function" style. But when I do, and try to run the tests it says:

![image](https://user-images.githubusercontent.com/36431357/141652089-9032e904-b36b-4909-bd73-b3d82d523e16.png)

And if I try to use require it's not defined. I try to use module.exports and it complains about needing to use export default.

Any ideas?

Reply:
@kibertoad So it's basically telling me/forcing me to use export default, and when I do it says no you can't do that...